tools/rollup-plugins/plugins.js

```diff
@@ -14,7 +14,7 @@ const plugins = (pkg, minify, esbuildOptions = {}) =>
     license({
       banner: `${pkg.name} v${pkg.version} - ${pkg.license}`,
     }),
-    bundleSize(),
+    // bundleSize(),
```

Comment:
Why remove this?

Reply:
Tnx, this was temporary. wiil place it back!